Libraries/Drei/base.lisp

```diff
@@ -275,7 +275,7 @@ constituent character of the line."
            until (end-of-buffer-p mark2)
            while (or (eql object #\Space) (eql object #\Tab))
            do (incf indentation
-                    (if (eql (object-after mark2) #\Tab) tab-width 1))
+                    (if (eql (object-after mark2) #\Tab) tab-width (/ tab-width 8)))
```

Comment:
what does it mean? if `tab-width` is 4 and character is not a tab we indent by 0.5? I don't understand how this could be right, if character is not a tab then indent is 1 like it was before.

Reply:
tab-space-count is 8 for a tab, other than that i don't know how or which other metrics go into the computation (like fonts or whatever), since there are many levels of abstraction like presentations/views/fonts etc..
the only thing we are sure is the tab-space-count. So what is 1 ?  is it 1 character or is it 1 in device units etc...
I thought that (/ tab-width 8) would give a more portable expression or even (/ tab-width tab-space-count) for that matter that's all, if you think it's really not needed or wrong then delete it please